pages/download/index.html

```diff
@@ -24,26 +24,26 @@ <h1>Download a Silverblue image</h1>
 advisable to keep the default partitioning setup that the Installer suggests,
 since rpm-ostree expects this setup.</p>
 <p> You can verify your download with
-<a href="https://download.fedoraproject.org/pub/fedora/linux/releases/30/Silverblue/x86_64/iso/Fedora-Silverblue-30-1.2-x86_64-CHECKSUM">
+<a href="https://download.fedoraproject.org/pub/fedora/linux/releases/31/Silverblue/x86_64/iso/Fedora-Silverblue-31-1.9-x86_64-CHECKSUM">
 this checksum file
 </a> by following
-<a href="https://docs.fedoraproject.org/en-US/fedora/f30/install-guide/install/Preparing_for_Installation/#sect-verifying-images">
+<a href="https://docs.fedoraproject.org/en-US/fedora/f31/install-guide/install/Preparing_for_Installation/#sect-verifying-images">
 these instructions
 <a>.
 </p>
 
-<h1>Try Fedora 31 Beta</h1>
+<h1>Try Fedora 32 Beta</h1>
```

Comment:
If we don't have F32 branded ISOs yet, let's just hide this whole section for now.

Reply:
Check the download link added "Coming Soon" there

Reply:
SGTM, I don't know if that's easy to "hide" or if we just need to delete the section.

Reply:
![image](https://user-images.githubusercontent.com/14056122/68043120-7a7c3880-fccc-11e9-8908-64470711aa93.png)

Reply:
@returntrip could we just comment out the whole section?